doc/source/whatsnew/v0.5.0.rst

```diff
@@ -1,7 +1,7 @@
 
 .. _whatsnew_050:
 
-v.0.5.0 (October 24, 2011)
+V.0.5.0 (October 24, 2011)
```

Comment:
I'd be consistent.
```suggestion
Version v.0.5.0 (October 24, 2011)
```

Reply:
Ah! Ok I didn't understand what you meant. Correcting